ingest-app/docs/api.md

```diff
@@ -1103,6 +1103,39 @@ Updated granules are validated using business rule validations.  Updates will no
 
 Granule bulk update currently supports updating with the following operations, update fields and metadata formats:
 
+**operation: "APPEND_TO_FIELD", update-field: "OPENDAPLink"**
```

Comment:
If `APPEND_TO_FIELD` is identical to `UPDATE_FIELD` for OPeNDAPLink, maybe we shouldn't support it for OPeNDAP Links?

Reply:
That seems to be the consensus and logical outcome of the use case. I'll amend